.goreleaser.yml

```diff
@@ -1,31 +1,53 @@
-# This is an example goreleaser.yaml file with some sane defaults.
-# Make sure to check the documentation at http://goreleaser.com
----
+version: 2
+
+project_name: revive
+
 archives:
   - id: revive
     name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    format: tar.gz
+
 before:
   hooks:
-    - "go mod download"
-    - "go generate ./..."
+    - go mod download
+    - go generate ./...
```

Comment:
Can we revert style changes that are not related to the topic of the PR?

Reply:
I propose to create two PRs:

1) the fix for https://github.com/mgechev/revive/issues/1104
2) the PR for updating GoReleaser configuration.